content/docs/atom-guide/look-dev/materials/material-type-file-spec.md

```diff
@@ -6,65 +6,479 @@ toc: true
 
+## **properties** (deprecated)
+An older version of the material type file format organized the property layout with separate `groups` and `properties` sections. This imposed several limitations like restricting property grouping to only one level and making it difficult to factor out reuasable property groups. This has been replaced by `propertyGroups` (above) which can support any number of group levels with properties and groups defined in the same place. The engine will still load the old format but any new material type files should use the new format.
 
-### **shaders**  
-An array of references to shader files (*.shader*). Each reference contains the following values. 
+## **shaders**  
+An array of references to shader files (`*.shader`) to be used to render materials of this type. By default, all shaders are enabled, but they may be disabled using [`functors`](#functors).
+
+Each shader item includes the following values. 
 
 * **file**: The path to the shader file. The path must be relative to the asset root or to the material type file.
-* **tag**: A unique name for this shader item that can be used to reference the shader from other places in the material type definition. It must be a C-Style identifier.
+* **tag**: A unique name for this shader item that can be used to reference the shader from other places in the material type definition, like in material functors. It must be a C-Style identifier.
+* **options**: Set the initial value for any options in this shader, using a list of key/value pairs.
 
 In this example, we reference the ShadowMap and DepthPass shaders. 
 ```JSON
 "shaders": [
     {
         "file": "../ShadowMap.shader",
-        "tag": "shadowmap"
+        "tag": "shadowmap",
+        "options": {
+            "o_depthBiasMode": "DepthBiasMode::Auto"
+        }
     },
     {
         "file": "../DepthPass.shader",
         "tag": "depth"
-    }
+    },
+    ...
 ]
 ```
 
-### **functor**
+## **functors**
 An array of material functors. Each one reads material property values, performs some logic or calculations, and sets shader inputs accordingly. These can be defined in Lua or C++. Each functor data contains the following:
 
 * **type**: The name of the functor type. This will be "Lua" for custom Lua script functors, or the name of specialized functor type (defined in C++).
+* **args**: An object containing key/value pairs of all the arguments to send to this functor. The arguments vary depending on functor type.
+
+Several core functor types are provided with the engine, listed here. 
+
+{{< note >}}
+Additional functor types can be added by other gems or game projects. Try searching the source code for "RegisterMaterialFunctor" to discover what others might be available.
+{{< /note >}}
+
+#### Lua
```

Comment:
Switching it to header 3 still maintains the hierarchy and allows the header to appear in the right-pane TOC navigation. 
```suggestion
### Lua
```